src/components/bark/login/layout.tsx

```diff
@@ -0,0 +1,70 @@
+import Image from "next/image";
+import { BarkH1 } from "../bark-typography";
+import { BarkNavRoute } from "../bark-nav";
+import Link from "next/link";
+
+const LoginLayout = ({
+  accountType,
+  children,
+}: Readonly<{ accountType: string; children: React.ReactNode }>) => {
+  let logoSrc = "";
+
+  if (accountType === "ADMIN") {
+    logoSrc = "/purpleDogHouse.svg";
+  } else if (accountType === "VET") {
+    logoSrc = "/pawPrint.svg";
+  } else if (accountType === "USER") {
+    logoSrc = "/orangeDogHouse.svg";
+  }
+
+  const footerRoutes: BarkNavRoute[] = [
+    {
+      label: "Contact us",
+      href: "/contact-us",
+    },
+    {
+      label: "Privacy Policy",
+      href: "/privacy-policy",
+    },
+    {
+      label: "Terms & Conditions",
+      href: "/terms-and-conditions",
```

Comment:
Would be nice to define more RoutePath for these. (In `route-path.ts`)

The reason is that while it's easy to move page.tsx about, it is not as easy to fix all the broken hrefs that references the moved page. But with a defined RoutePath we can redefine the href in one place.

Reply:
Done 59f4ca286cdb1a0c1352c48a31daccf6c000ab5c